estuary_updater/handlers/base.py

```diff
@@ -105,9 +123,14 @@ def get_or_create_build(self, identifier, original_nvr=None, force_container_lab
             log.error('Failed to get brew build using the identifier {0}'.format(identifier))
             raise
 
+        try:
+            extra_json = json.dumps(build_info['extra'])
```

Comment:
Are you sure you need to do this? I thought that the Koji API was returning data already formatted in JSON

Reply:
This is correct because we want to convert extra (currently a dict) to a string.